operator/src/main/java/org/bf2/operator/operands/KafkaCluster.java

```diff
@@ -618,7 +624,14 @@ private KafkaExporterSpec buildKafkaExporter(ManagedKafka managedKafka) {
                 .withTopicRegex(".*")
                 .withGroupRegex(".*")
                 .withImage(this.overrideManager.getKafkaExporterImage(strimzi).orElse(null))
-                .withResources(config.getExporter().buildResources());
+                .withResources(config.getExporter().buildResources())
+                .withNewTemplate()
+                .withNewPod()
+                .withNewMetadata()
+                .withAnnotations(SAFE_TO_EVICT)
+                .endMetadata()
+                .endPod()
+                .endTemplate();
```

Comment:
Just confirming - Exporter and Cruise Control are backed by `Deployment`s, so this is due to local storage?
https://github.com/kubernetes/autoscaler/blob/master/cluster-autoscaler/FAQ.md#what-types-of-pods-can-prevent-ca-from-removing-a-node

Reply:
Yes, see the pods listed under https://issues.redhat.com/browse/MGDSTRM-9741